### PR TITLE
Update upgrade docs

### PR DESCRIPTION
**Pull request by dreamer-zq**

### CODE REVIEW

File: docs/zh/features/upgrade.md [hunk 1]
```diff
@@ -103,3 +103,39 @@ type SoftwareUpgradeProposal struct {
...
+软件升级提案和其他普通提案的执行流程基本一致，都需要验证人、委托人为该提案发表意见，具体信息请参考[治理模块](./governance.md)。为提案抵押的命令行示例如下：
+
+```bash
+iris tx gov deposit 1 1000000000uiris --from=node0 --chain-id=test --fees=1000000uiris -b block -y
+```
+
+一旦抵押金额达到最小抵押金额，提案将进入投票期，验证人或者委托人需要对该提案发起投票，发起投票的命令行示例如下：
+
+```bash
+iris tx gov vote 1 yes  --from=node0 --chain-id=test 1000000uiris -b block -y
```
Comment (zhangyelong) [inline docs/zh/features/upgrade.md]:
```suggestion
iris tx gov vote <proposal_id> <yes/no/no_with_veto/abstain> --from=<key-name> --chain-id=<chain-id> --fees=0.3iris -b block -y
```

File: docs/zh/features/upgrade.md [hunk 2]
```diff
@@ -103,3 +103,39 @@ type SoftwareUpgradeProposal struct {
...
+执行软件升级流程的第一步是由治理模块发起一个软件升级提案，该提案详细说明了升级时间以及升级内容，具体见上面[概念](#概念)。发起提案的命令行示例如下：
+
+```bash
+iris tx gov submit-proposal software-upgrade bifrost-rc2 \
+  --deposit 10000000uiris \
+  --upgrade-time 2021-02-09T13:00:00Z \
+  --title "mainnet software upgrade" \
+  --upgrade-info "Commit: 0ef5dd0b4d140a4788f05fc1a0bd409b3c6a0492. After the proposal is approved, please use the commit hash to build and restart your node." \
+  --description "Upgrade the mainnet software version from v1.0.0-rc0 to v1.0.0-rc2."
+  --from=node0 --chain-id=test 1000000uiris -b block -y
```
Comment (zhangyelong) [inline docs/zh/features/upgrade.md]:
```suggestion
  --from=<key-name> --chain-id=<chain-id> --fees=0.3iris -b block -y
```

File: docs/zh/features/upgrade.md [hunk 3]
```diff
@@ -103,3 +103,39 @@ type SoftwareUpgradeProposal struct {
...
+```
+
+### 为提案抵押、投票
+
+软件升级提案和其他普通提案的执行流程基本一致，都需要验证人、委托人为该提案发表意见，具体信息请参考[治理模块](./governance.md)。为提案抵押的命令行示例如下：
+
+```bash
+iris tx gov deposit 1 1000000000uiris --from=node0 --chain-id=test --fees=1000000uiris -b block -y
```
Comment (zhangyelong) [inline docs/zh/features/upgrade.md]:
```suggestion
iris tx gov deposit <proposal-id> 100iris --from=<key-name> --chain-id=<chain-id> --fees=0.3iris -b block -y
```

File: docs/zh/features/upgrade.md [hunk 4]
```diff
@@ -103,3 +103,39 @@ type SoftwareUpgradeProposal struct {
...
+执行软件升级流程的第一步是由治理模块发起一个软件升级提案，该提案详细说明了升级时间以及升级内容，具体见上面[概念](#概念)。发起提案的命令行示例如下：
+
+```bash
+iris tx gov submit-proposal software-upgrade bifrost-rc2 \
+  --deposit 1000iris \
+  --upgrade-time 2021-02-09T13:00:00Z \
+  --title "mainnet software upgrade" \
+  --upgrade-info "Commit: 0ef5dd0b4d140a4788f05fc1a0bd409b3c6a0492. After the proposal is approved, please use the commit hash to build and restart your node." \
+  --description "Upgrade the mainnet software version from v1.0.0-rc0 to v1.0.0-rc2."
+  --from=node0 --chain-id=test --fees=1iris -b block -y
```
Comment (zhangyelong) [inline docs/zh/features/upgrade.md:120]:
```suggestion
  --from=<key-name> --chain-id=<chain-id> --fees=0.3iris -b block -y
```

File: docs/zh/features/upgrade.md [hunk 5]
```diff
@@ -103,3 +103,39 @@ type SoftwareUpgradeProposal struct {
...
+```
+
+### 为提案抵押、投票
+
+软件升级提案和其他普通提案的执行流程基本一致，都需要验证人、委托人为该提案发表意见，具体信息请参考[治理模块](./governance.md)。为提案抵押的命令行示例如下：
+
+```bash
+iris tx gov deposit 1 1000iris --from=node0 --chain-id=test --fees=1iris -b block -y
```
Comment (zhangyelong) [inline docs/zh/features/upgrade.md:128]:
```suggestion
iris tx gov deposit <proposal-id> 1000iris --from=<key-name> --chain-id=<chain-id> --fees=0.3iris -b block -y
```

File: docs/zh/features/upgrade.md [hunk 6]
```diff
@@ -103,3 +103,39 @@ type SoftwareUpgradeProposal struct {
...
+```
+
+一旦抵押金额达到最小抵押金额，提案将进入投票期，验证人或者委托人需要对该提案发起投票，发起投票的命令行示例如下：
+
+```bash
+iris tx gov vote 1 yes  --from=node0 --chain-id=test --fees=1iris -b block -y
```
Comment (zhangyelong) [inline docs/zh/features/upgrade.md:134]:
```suggestion
iris tx gov vote <proposal-id> <yes/no/no_with_veto/abstain> --from=<key-name> --chain-id=<chain-id> --fees=0.3iris -b block -y
```

File: docs/features/upgrade.md [hunk 7]
```diff
@@ -107,3 +107,39 @@ The user replaces the software with the specified version and restarts the netwo
...
+The first step in the implementation of the software upgrade process is to initiate a software upgrade proposal by the governance module. The proposal details the upgrade time and upgrade content. For details, see the above [Concept](#Concepts). An example of a command line to initiate a proposal is as follows:
+
+```bash
+iris tx gov submit-proposal software-upgrade bifrost-rc2 \
+  --deposit 1000iris \
+  --upgrade-time 2021-02-09T13:00:00Z \
+  --title "mainnet software upgrade" \
+  --upgrade-info "Commit: 0ef5dd0b4d140a4788f05fc1a0bd409b3c6a0492. After the proposal is approved, please use the commit hash to build and restart your node." \
+  --description "Upgrade the mainnet software version from v1.0.0-rc0 to v1.0.0-rc2."
+  --from=node0 --chain-id=test --fees=1iris -b block -y
```
Comment (SegueII) [inline docs/features/upgrade.md:124]: will it be better if the parameters were not filled?

File: docs/zh/features/upgrade.md [hunk 8]
```diff
@@ -103,3 +103,39 @@ type SoftwareUpgradeProposal struct {
...
+```
+
+当软件升级提案被通过后，升级模块会创建一项升级计划，在指定高度或者时间使所有节点停止网络共识，等待新的软件重启网络。
+
+### 重启网络
+
+当升级提案通过，节点会停止出块，用户需要根据在第一步[提交升级提案](#提交升级提案)中指明的新版本信息下载源码，编译新软件，具体参考[安装](./../get-started/install.md)。新软件安装完成后，使用新的版本重启节点，节点会执行和计划名称相应的升级逻辑。一旦全网超过2/3的权重使用新的版本重启网络，区块链网络将重新达成新的共识，继续产生新区块。
```
Comment (zhangyelong) [inline docs/zh/features/upgrade.md:141]:
```suggestion
当升级提案通过，网络到达指定升级区块高度或时间时，所有节点会停止出块。用户需要根据在第一步[提交升级提案](#提交升级提案)中指明的新版本信息下载源码，编译新软件，具体参考[安装](./../get-started/install.md)。新软件安装完成后，使用新的版本重启节点。当全网超过2/3的权重的验证人使用新版本启动节点后，区块链网络将重新达成新的共识，继续产生区块。
```

File: docs/zh/features/upgrade.md [hunk 9]
```diff
@@ -103,3 +103,39 @@ type SoftwareUpgradeProposal struct {
...
 如果当前升级计划已经被执行，但是该升级计划存在问题，那么此时提出`取消软件升级`提议是无效的(因为网络已经停止共识)。这时还有另一方案来弥补这一过失，就是在重新启动网络时，使用`--unsafe-skip-upgrades`参数跳过指定的升级高度(并不是真的跳过该高度，而是跳过软件升级`处理器`)。当然这要求参与共识的2/3验证人都执行同样的操作，否则同样无法达成网络共识。
+
+## 升级流程
+
+### 提交升级提案
+
+执行软件升级流程的第一步是由治理模块发起一个软件升级提案，该提案详细说明了升级时间以及升级内容，具体见上面[概念](#概念)。发起提案的命令行示例如下：
+
+```bash
+iris tx gov submit-proposal software-upgrade bifrost-rc2 \
```
Comment (zhangyelong) [inline docs/zh/features/upgrade.md:114]:
```suggestion
iris tx gov submit-proposal software-upgrade <upgrade-name> \
```

**Review by zhangyelong — CHANGES_REQUESTED** [review]

**Review by chengwenxi — CHANGES_REQUESTED** [review]

**Review by SegueII — APPROVED** [review]

**Review by zhangyelong — APPROVED** [review]

**Review by chengwenxi — APPROVED** [review]

**Pull request merged**